docs/user/upgrade.rst

```diff
@@ -0,0 +1,70 @@
+Upgrading
+=========
+
+.. warning:: These instructions are under development
+
+Upgrade paths
+-------------
+
+Kolibri can be automatically upgraded forwards. For instance, you can upgrade
+from ``0.1->0.2`` and ``0.1->0.7``. We test all upgrade paths, but we also
+caution that the more versions that you skip, the higher the risks will be
+that something isn't working as expected.
+
+That's why we also support :ref:`downgrading`.
+
+Every time Kolibri is upgraded, it will automatically migrate your database
+and create a backup before doing so.
+
+.. note:: Always upgrade **as often as possible**. If you are responsible for
+  many installations, you should 
```

Comment:
looks like something here was started but not finished

Reply:
probably can leave out this guidance altogether

Reply:
Thanks, I've finalized the sentence - I don't mind that it gets moved or deleted later on, just wanted to share the idea that we encourage people to have an upgrade strategy of some sort (the missing part of the sentence).